torchci/rockset/inductor/__sql/compilers_benchmark_performance.sql

```diff
@@ -9,6 +9,7 @@ WITH performance_results AS (
         '_performance'
       )
     ) AS filename,
+    abs_latency,
```

Comment:
For this file, you would need to update the query on Rockset https://console.rockset.com/lambdas/details/inductor.compilers_benchmark_performance and update the version in https://github.com/pytorch/test-infra/blob/main/torchci/rockset/prodVersions.json#L79.

Then, tou can either update the local SQL file manually or run `node scripts/downloadQueryLambda.mjs inductor compilers_benchmark_performance` to sync them.  The local file needs to be exactly the same as the specified version on Rockset to have effect.

Reply:
I can see that the new column has shown up https://torchci-git-desertfire-perf-dashboard-fbopensource.vercel.app/benchmark/torchbench/inductor_with_cudagraphs?startTime=Thu,%2018%20May%202023%2023:19:09%20GMT&stopTime=Thu,%2025%20May%202023%2023:19:09%20GMT&mode=training&dtype=amp&lBranch=main&lCommit=a896962f0aa78a037513dac4fa60abd14adcb0ff&rBranch=main&rCommit=e06bd8f3b11e8843596931514f59579e47ae7b12, but no data there yet